doc/user/content/releases/v0.36.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Materialize v0.36"
+date: 2022-12-14
+released: true
+---
+
+## v0.36.0
+
+* Add `mz_internal.mz_sink_status` and `mz_internal.mz_sink_status_history`
+  to the system catalog. These objects respectively expose the current and
+  historical state for each sink in the system, including potential error
+  messages and additional metadata helpful for debugging.
+
+* Add `mz_internal.mz_cluster_replica_sizes` to the system catalog. This table
+  provides a mapping of logical sizes (e.g. `xlarge`) to the number of
+  processes, as well as CPU and memory allocations for each process. To monitor
+  the resource utilization for all extant cluster replicas as a % of the total
+  allocation, you can now use:
+
+  ```sql
+  SELECT
+    r.id AS replica_id,
+    m.process_id,
+    m.cpu_nano_cores / s.cpu_nano_cores * 100 AS cpu_percent,
+    m.memory_bytes / s.memory_bytes * 100 AS memory_percent
+  FROM mz_cluster_replicas AS r
+  JOIN mz_internal.mz_cluster_replica_sizes AS s ON r.size = s.size
+  JOIN mz_internal.mz_cluster_replica_metrics AS m ON m.replica_id = r.id;
+  ```
+
+* Add `mz_catalog.mz_aws_privatelink_connections` to the system catalog. This
+  table contains a row for each [AWS PrivateLink connection](/sql/create-connection/#aws-privatelink)
+  in the system, and allows you to retrieve the AWS principal that
+  Materialize will use to connect to the VPC endpoint.
+
+* Return an error rather than crashing if the value of the `AVRO KEY FULLNAME`
+  or `AVRO VALUE FULLNAME` option in an Avro-formatted Kafka sink is not a
+  valid Avro name {{% gh 16433 %}}.
+
+* Return the current timestamp of the `EpochMillis` timeline when the `mz_now
+  ()` function is used outside the context of a specific timeline, such as
```

Comment:
Have we ever mentioned "timeline" in the docs before? I wonder if we should just say

"Return the current system time when the `mz_now()` function is used in a query that does not refer to any objects, such as `SELECT mz_now()`."

cc @jkosh44

Reply:
We have, but I'd much rather use your suggestion! Got tied up in the details of the distinction between timeline -in dependent. And sleep!

Reply:
We do have a [docs page on timelines](https://github.com/MaterializeInc/materialize/blob/main/doc/user/content/overview/timelines.md)! But it was marked as draft when we switched to cloud because it focused mostly on Materialize CDC.